Initial documentation review: getting-started (#303)
egil/bUnit: docs/site/docs/test-doubles/faking-auth.md

- **Authenticated** and **authorized** 
- **Authenticated** and **authorized** with one or more **roles**, **claims**, and/or **policies**

bUnit's authentication and authorization implementation is easily available by calling [`AddTestAuthorization()`](xref:Bunit.TestDoubles.FakeAuthorizationExtensions.AddTestAuthorization(Bunit.TestContextBase)) on a test context. This adds the necessary services to the `Services` collection and the `CascadingAuthenticationState` component to the [root render tree](xref:root-render-tree). The method returns an instance of the <xref:Bunit.TestDoubles.TestAuthorizationContext> type that allows you to control the authentication and authorization state for a test.

> [!NOTE]

> > If your [__AP: Is there a noun missing here?__] inherits directly from bUnit's <xref:Bunit.TestContext>, as described in <xref:writing-csharp-tests#remove-boilerplate-code-from-tests>, then you need to call the [`AddTestAuthorization()`](xref:Bunit.TestDoubles.FakeAuthorizationExtensions.AddTestAuthorization(Bunit.TestContextBase)) method like so: `this.AddTestAuthorization()`.

Reviewer: ```suggestion
> If your test class inherits directly from bUnit's <xref:Bunit.TestContext>, as described in <xref:writing-csharp-tests#remove-boilerplate-code-from-tests>, then you need to call the [`AddTestAuthorization()`](xref:Bunit.TestDoubles.FakeAuthorizationExtensions.AddTestAuthorization(Bunit.TestContextBase)) method like so: `this.AddTestAuthorization()`.
```
> [__AP: Is there a noun missing here?__]

bUnit's authentication and authorization implementation is easily available by calling [`AddTestAuthorization()`](xref:Bunit.TestDoubles.FakeAuthorizationExtensions.AddTestAuthorization(Bunit.TestContext)) on a test context. This adds the necessary services to the `Services` collection and the `CascadingAuthenticationState` component to the [root render tree](xref:root-render-tree). The method returns an instance of the <xref:Bunit.TestDoubles.TestAuthorizationContext> type that allows you to control the authentication and authorization state for a test.

> [!NOTE]

> > If your [__AP: Is there a noun missing here?__] inherits directly from bUnit's <xref:Bunit.TestContext>, as described in <xref:writing-csharp-tests#remove-boilerplate-code-from-tests>, then you need to call the [`AddTestAuthorization()`](xref:Bunit.TestDoubles.FakeAuthorizationExtensions.AddTestAuthorization(Bunit.TestContext)) method like so: `this.AddTestAuthorization()`.

Reviewer: ```suggestion
> If you inherit directly from bUnit's <xref:Bunit.TestContext>, as described in <xref:writing-csharp-tests#remove-boilerplate-code-from-tests>, then you need to call the [`AddTestAuthorization()`](xref:Bunit.TestDoubles.FakeAuthorizationExtensions.AddTestAuthorization(Bunit.TestContext)) method like so: `this.AddTestAuthorization()`.
```

No noun missing, just bad spelling. This looks right.

Author: Ah yes, that's better!

The following sections show how to set each of these states in a test.

## Setting Authenticating, Authenticated and Authorized States

> The examples in the following sections will use the `<UserInfo>` component listed below. This uses an injected `AuthenticationStateProvider` service  and `<CascadingAuthenticationState>` and `<AuthorizeView>` components to show the user name when a user is authenticated. It also shows the authorization state when the authenticated user is authorized.

Reviewer: ```suggestion
The examples in the following sections will use the `<UserInfo>` component listed below. This uses an injected `AuthenticationStateProvider` service and `<CascadingAuthenticationState>` and `<AuthorizeView>` components to show the user name when a user is authenticated. It also shows the authorization state when the authenticated user is authorized.
```

One space too many

[!code-csharp[UserRightsTest.cs](../../../samples/tests/xunit/UserRightsTest.cs?start=106&end=123&highlight=5-8)]

> The highlighted line shows how the <xref:Bunit.TestDoubles.TestAuthorizationContext.SetClaims(System.Security.Claims.Claim[])> method is used to pass two instances of the `Claim` types.[__AP: type or types?__]

Reviewer: ```suggestion
The highlighted line shows how the <xref:Bunit.TestDoubles.TestAuthorizationContext.SetClaims(System.Security.Claims.Claim[])> method is used to pass two instances of the `Claim` type.
```

> [__AP: type or types?__]
